org.talend.selenium.tac.test/src/main/java/org/talend/tac/fileFolder/administrator_config.txt
config.scheduler,config.scheduler.archiveAndExecutionLogsPath,/home/automation/Automation/automation/filefolder/scheduler/archiveJobs,true
config.scheduler,config.scheduler.recoveryLogsPath,/home/automation/Automation/automation/filefolder/scheduler/executionLogs,true
config.scheduler,config.scheduler.quarzServlet,Initialized,true
config.primary.commandLine,config.commandLine.host,192.168.0.39,true
config.primary.commandLine,config.commandLine.port,8002,true
config.primary.commandLine,config.commandLine.archivePath,/home/automation/Automation/automation/filefolder/commandline,null
config.primary.commandLine,config.commandLine.userComponentPath,,null
config.secondary.commandLine,config.commandLine.host,localhost,true
config.secondary.commandLine,config.commandLine.port,8003,false
config.secondary.commandLine,config.commandLine.archivePath,c:/Talend/CommandLine/exports,null
config.secondary.commandLine,config.commandLine.userComponentPath,,null
config.database,config.database.url,jdbc:h2:/home/automation/hudson_selenium/apache-tomcat/webapps/org.talend.administrator/WEB-INF/database/talend_administrator;AUTO_SERVER=TRUE;MVCC=TRUE,true
config.database,config.database.user,tisadmin,true
config.database,config.database.password,tisadmin,true
config.database,config.database.driver,org.h2.Driver,true
config.database,config.database.link,<a href="http://192.168.0.39:8080//org.talend.administrator/h2console" target="_blank">http://192.168.0.39:8080//org.talend.administrator/h2console</a>,true
config.log4j,config.log4j.talendAppender,/home/automation/Automation/automation/filefolder/talendappender.log,true
config.log4j,config.log4j.talendAppender.threshold,WARN,true
config.general,config.general.webAppVersion,5.0.0NB.64106,null
config.general,config.general.dbModelVersion,5.0.0NB.64106,null
config.general,config.general.location,http://192.168.0.39:8080//org.talend.administrator,true
config.soaManager,config.soaManager.jobsDeployFolder,/home/automation/Automation/automation/filefolder/soa,true
config.soaManager,config.soaManager.defaultServiceNameSpace,http://www.talend.org,null
config.soaManager,config.soaManager.host,192.168.0.39,true
config.svn,config.svn.serverLocationUrl,https://192.168.0.216:8443/svn/test_tac,true
config.svn,config.svn.username,admin,true
config.svn,config.svn.password,admin,true
config.soaManager,config.soaManager.port,8004,true
config.smtp,config.smtp.useSmtp,false,true
config.smtp,config.smtp.host,smtp.163.com,true
config.smtp,config.smtp.port,25,true
config.smtp,config.smtp.requireSSL,false,true
config.smtp,config.smtp.userName,lindy512@163.com,true
config.smtp,config.smtp.password,talend,true
config.ldap,config.ldap.useLDAPAutentication,false,null
config.ldap,config.ldap.host,192.168.0.27,null
config.ldap,config.ldap.port,389,null
config.ldap,config.ldap.root,dc=talend,dc=com,null
config.ldap,config.ldap.principalDNPrefix,cn=Manager,dc=talend,dc=com,null
config.ldap,config.ldap.adminPassword,talend,null
config.ldap,config.ldap.emailField,mail,null
config.ldap,config.ldap.firstName,givenName,null
config.ldap,config.ldap.lastName,homePhone,null
label.suite,menu.dqportal,,null
label.suite,menu.drools,,null
label.suite,menu.mdm,,null
dynamic.link,menu.dynamicLink.url,,null
----------------------------------------------------
Exported at Mon Jun 27 15:04:31 CST 2011
----------------------------------------------------
java.runtime.name=Java(TM) SE Runtime Environment
sun.boot.library.path=/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/i386
java.vm.version=19.1-b02
shared.loader=
java.vm.vendor=Sun Microsystems Inc.
java.vendor.url=http://java.sun.com/
path.separator=:
java.vm.name=Java HotSpot(TM) Server VM
tomcat.util.buf.StringCache.byte.enabled=true
file.encoding.pkg=sun.io
user.country=US
sun.java.launcher=SUN_STANDARD
sun.os.patch.level=unknown
java.vm.specification.name=Java Virtual Machine Specification
user.dir=/home/automation/Automation/automation/selenium_tomcat/bin
java.runtime.version=1.6.0_24-b07
java.awt.graphicsenv=sun.awt.X11GraphicsEnvironment
java.endorsed.dirs=/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/endorsed
os.arch=i386
java.io.tmpdir=/tmp
line.separator=

java.vm.specification.vendor=Sun Microsystems Inc.
java.naming.factory.url.pkgs=org.apache.naming
os.name=Linux
sun.jnu.encoding=UTF-8
java.library.path=/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/i386/server:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/i386:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/../lib/i386:/usr/java/packages/lib/i386:/lib:/usr/lib
java.specification.name=Java Platform API Specification
java.class.version=50.0
sun.management.compiler=HotSpot Tiered Compilers
os.version=2.6.35-28-generic
user.home=/home/automation
user.timezone=Asia/Chongqing
catalina.useNaming=true
java.awt.printerjob=sun.print.PSPrinterJob
java.specification.version=1.6
file.encoding=UTF-8
catalina.home=/home/automation/Automation/automation/selenium_tomcat
user.name=automation
java.class.path=bootstrap.jar
java.naming.factory.initial=org.apache.naming.java.javaURLContextFactory
package.definition=sun.,java.,org.apache.catalina.,org.apache.coyote.,org.apache.tomcat.,org.apache.jasper.
java.vm.specification.version=1.0
sun.arch.data.model=32
java.home=/usr/lib/jvm/java-6-sun-1.6.0.24/jre
java.specification.vendor=Sun Microsystems Inc.
user.language=en
java.vm.info=mixed mode
java.version=1.6.0_24
java.ext.dirs=/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/ext:/usr/java/packages/lib/ext
sun.boot.class.path=/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/resources.jar:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/rt.jar:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/jsse.jar:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/jce.jar:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/charsets.jar:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/lib/modules/jdk.boot.jar:/usr/lib/jvm/java-6-sun-1.6.0.24/jre/classes
java.vendor=Sun Microsystems Inc.
server.loader=
catalina.base=/home/automation/Automation/automation/selenium_tomcat
file.separator=/
java.vendor.url.bug=http://java.sun.com/cgi-bin/bugreport.cgi
common.loader=${catalina.base}/lib,${catalina.base}/lib/*.jar,${catalina.home}/lib,${catalina.home}/lib/*.jar
sun.cpu.endian=little
sun.io.unicode.encoding=UnicodeLittle
package.access=sun.,org.apache.catalina.,org.apache.coyote.,org.apache.tomcat.,org.apache.jasper.,sun.beans.
sun.desktop=gnome
sun.cpu.isalist=

org.talend.model.migration/transformation/development/Config.asm
<?xml version = '1.0' encoding = 'ISO-8859-1' ?>
<asm name="0">
	<cp>
		<constant value="Config"/>
		<constant value="main"/>
		<constant value="A"/>
		<constant value="String"/>
		<constant value="#native"/>
		<constant value="path"/>
		<constant value="__initpath"/>
		<constant value="J.registerHelperAttribute(SS):V"/>
		<constant value="ecore::EClass"/>
		<constant value="ECORE"/>
		<constant value="inclusionCondition"/>
		<constant value="__initinclusionCondition"/>
		<constant value="ecore::ENamedElement"/>
		<constant value="useQualifiedName"/>
		<constant value="__inituseQualifiedName"/>
		<constant value="useQualifiedRuleName"/>
		<constant value="__inituseQualifiedRuleName"/>
		<constant value="copyXmiID"/>
		<constant value="__initcopyXmiID"/>
		<constant value="useIsInElementHelper"/>
		<constant value="__inituseIsInElementHelper"/>
		<constant value="6:16-6:22"/>
		<constant value="10:16-10:37"/>
		<constant value="30:16-30:44"/>
		<constant value="34:16-34:44"/>
		<constant value="38:16-38:37"/>
		<constant value="43:16-43:37"/>
		<constant value="self"/>
		<constant value="S"/>
		<constant value="/org.talend.model.migration/transformation/development/"/>
		<constant value="0"/>
		<constant value="J.+(J):J"/>
		<constant value="7:2-7:59"/>
		<constant value="7:62-7:66"/>
		<constant value="7:2-7:66"/>
		<constant value="MECORE!ecore::EClass;"/>
		<constant value="name"/>
		<constant value="EAttribute"/>
		<constant value="J.=(J):J"/>
		<constant value="EClass"/>
		<constant value="J.or(J):J"/>
		<constant value="EDataType"/>
		<constant value="EEnum"/>
		<constant value="EEnumLiteral"/>
		<constant value="EFactory"/>
		<constant value="EObject"/>
		<constant value="EOperation"/>
		<constant value="EPackage"/>
		<constant value="EParameter"/>
		<constant value="EReference"/>
		<constant value="EGenericType"/>
		<constant value="ETypeParameter"/>
		<constant value="Profile"/>
		<constant value="Stereotype"/>
		<constant value="J.not():J"/>
		<constant value="12:6-12:10"/>
		<constant value="12:6-12:15"/>
		<constant value="12:18-12:30"/>
		<constant value="12:6-12:30"/>
		<constant value="13:6-13:10"/>
		<constant value="13:6-13:15"/>
		<constant value="13:18-13:26"/>
		<constant value="13:6-13:26"/>
		<constant value="12:6-13:26"/>
		<constant value="14:3-14:7"/>
		<constant value="14:3-14:12"/>
		<constant value="14:15-14:26"/>
		<constant value="14:3-14:26"/>
		<constant value="12:6-14:26"/>
		<constant value="15:3-15:7"/>
		<constant value="15:3-15:12"/>
		<constant value="15:15-15:22"/>
		<constant value="15:3-15:22"/>
		<constant value="12:6-15:22"/>
		<constant value="16:3-16:7"/>
		<constant value="16:3-16:12"/>
		<constant value="16:15-16:29"/>
		<constant value="16:3-16:29"/>
		<constant value="12:6-16:29"/>
		<constant value="17:3-17:7"/>
		<constant value="17:3-17:12"/>
		<constant value="17:15-17:25"/>
		<constant value="17:3-17:25"/>
		<constant value="12:6-17:25"/>
		<constant value="18:3-18:7"/>
		<constant value="18:3-18:12"/>
		<constant value="18:15-18:24"/>
		<constant value="18:3-18:24"/>
		<constant value="12:6-18:24"/>
		<constant value="19:3-19:7"/>
		<constant value="19:3-19:12"/>
		<constant value="19:15-19:27"/>
		<constant value="19:3-19:27"/>
		<constant value="12:6-19:27"/>
		<constant value="20:3-20:7"/>
		<constant value="20:3-20:12"/>
		<constant value="20:15-20:25"/>
		<constant value="20:3-20:25"/>
		<constant value="12:6-20:25"/>
		<constant value="21:3-21:7"/>
		<constant value="21:3-21:12"/>
		<constant value="21:15-21:27"/>
		<constant value="21:3-21:27"/>
		<constant value="12:6-21:27"/>
		<constant value="22:3-22:7"/>
		<constant value="22:3-22:12"/>
		<constant value="22:15-22:27"/>
		<constant value="22:3-22:27"/>
		<constant value="12:6-22:27"/>
		<constant value="23:3-23:7"/>
		<constant value="23:3-23:12"/>
		<constant value="23:15-23:29"/>
		<constant value="23:3-23:29"/>
		<constant value="12:6-23:29"/>
		<constant value="24:3-24:7"/>
		<constant value="24:3-24:12"/>
		<constant value="24:15-24:31"/>
		<constant value="24:3-24:31"/>
		<constant value="12:6-24:31"/>
		<constant value="25:3-25:7"/>
		<constant value="25:3-25:12"/>
		<constant value="25:15-25:24"/>
		<constant value="25:3-25:24"/>
		<constant value="12:6-25:24"/>
		<constant value="26:3-26:7"/>
		<constant value="26:3-26:12"/>
		<constant value="26:15-26:27"/>
		<constant value="26:3-26:27"/>
		<constant value="12:6-26:27"/>
		<constant value="11:2-27:3"/>
		<constant value="MECORE!ecore::ENamedElement;"/>
		<constant value="31:2-31:6"/>
		<constant value="35:2-35:6"/>
		<constant value="39:2-39:6"/>
		<constant value="44:2-44:7"/>
	</cp>
	<operation name="1">
		<context type="2"/>
		<parameters>
		</parameters>
		<code>
			<push arg="3"/>
			<push arg="4"/>
			<findme/>
			<push arg="5"/>
			<push arg="6"/>
			<call arg="7"/>
			<push arg="8"/>
			<push arg="9"/>
			<findme/>
			<push arg="10"/>
			<push arg="11"/>
			<call arg="7"/>
			<push arg="12"/>
			<push arg="9"/>
			<findme/>
			<push arg="13"/>
			<push arg="14"/>
			<call arg="7"/>
			<push arg="12"/>
			<push arg="9"/>
			<findme/>
			<push arg="15"/>
			<push arg="16"/>
			<call arg="7"/>
			<push arg="8"/>
			<push arg="9"/>
			<findme/>
			<push arg="17"/>
			<push arg="18"/>
			<call arg="7"/>
			<push arg="8"/>
			<push arg="9"/>
			<findme/>
			<push arg="19"/>
			<push arg="20"/>
			<call arg="7"/>
		</code>
		<linenumbertable>
			<lne id="21" begin="0" end="2"/>
			<lne id="22" begin="6" end="8"/>
			<lne id="23" begin="12" end="14"/>
			<lne id="24" begin="18" end="20"/>
			<lne id="25" begin="24" end="26"/>
			<lne id="26" begin="30" end="32"/>
		</linenumbertable>
		<localvariabletable>
			<lve slot="0" name="27" begin="0" end="35"/>
		</localvariabletable>
	</operation>
	<operation name="6">
		<context type="28"/>
		<parameters>
		</parameters>
		<code>
			<push arg="29"/>
			<load arg="30"/>
			<call arg="31"/>
		</code>
		<linenumbertable>
			<lne id="32" begin="0" end="0"/>
			<lne id="33" begin="1" end="1"/>
			<lne id="34" begin="0" end="2"/>
		</linenumbertable>
		<localvariabletable>
			<lve slot="0" name="27" begin="0" end="2"/>
		</localvariabletable>
	</operation>
	<operation name="11">
		<context type="35"/>
		<parameters>
		</parameters>
		<code>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="37"/>
			<call arg="38"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="39"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="41"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="42"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="43"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="44"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="45"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="46"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="47"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="48"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="49"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="50"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="51"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="52"/>
			<call arg="38"/>
			<call arg="40"/>
			<load arg="30"/>
			<get arg="36"/>
			<push arg="53"/>
			<call arg="38"/>
			<call arg="40"/>
			<call arg="54"/>
		</code>
		<linenumbertable>
			<lne id="55" begin="0" end="0"/>
			<lne id="56" begin="0" end="1"/>
			<lne id="57" begin="2" end="2"/>
			<lne id="58" begin="0" end="3"/>
			<lne id="59" begin="4" end="4"/>
			<lne id="60" begin="4" end="5"/>
			<lne id="61" begin="6" end="6"/>
			<lne id="62" begin="4" end="7"/>
			<lne id="63" begin="0" end="8"/>
			<lne id="64" begin="9" end="9"/>
			<lne id="65" begin="9" end="10"/>
			<lne id="66" begin="11" end="11"/>
			<lne id="67" begin="9" end="12"/>
			<lne id="68" begin="0" end="13"/>
			<lne id="69" begin="14" end="14"/>
			<lne id="70" begin="14" end="15"/>
			<lne id="71" begin="16" end="16"/>
			<lne id="72" begin="14" end="17"/>
			<lne id="73" begin="0" end="18"/>
			<lne id="74" begin="19" end="19"/>
			<lne id="75" begin="19" end="20"/>
			<lne id="76" begin="21" end="21"/>
			<lne id="77" begin="19" end="22"/>
			<lne id="78" begin="0" end="23"/>
			<lne id="79" begin="24" end="24"/>
			<lne id="80" begin="24" end="25"/>
			<lne id="81" begin="26" end="26"/>
			<lne id="82" begin="24" end="27"/>
			<lne id="83" begin="0" end="28"/>
			<lne id="84" begin="29" end="29"/>
			<lne id="85" begin="29" end="30"/>
			<lne id="86" begin="31" end="31"/>
			<lne id="87" begin="29" end="32"/>
			<lne id="88" begin="0" end="33"/>
			<lne id="89" begin="34" end="34"/>
			<lne id="90" begin="34" end="35"/>
			<lne id="91" begin="36" end="36"/>
			<lne id="92" begin="34" end="37"/>
			<lne id="93" begin="0" end="38"/>
			<lne id="94" begin="39" end="39"/>
			<lne id="95" begin="39" end="40"/>
			<lne id="96" begin="41" end="41"/>
			<lne id="97" begin="39" end="42"/>
			<lne id="98" begin="0" end="43"/>
			<lne id="99" begin="44" end="44"/>
			<lne id="100" begin="44" end="45"/>
			<lne id="101" begin="46" end="46"/>
			<lne id="102" begin="44" end="47"/>
			<lne id="103" begin="0" end="48"/>
			<lne id="104" begin="49" end="49"/>
			<lne id="105" begin="49" end="50"/>
			<lne id="106" begin="51" end="51"/>
			<lne id="107" begin="49" end="52"/>
			<lne id="108" begin="0" end="53"/>
			<lne id="109" begin="54" end="54"/>
			<lne id="110" begin="54" end="55"/>
			<lne id="111" begin="56" end="56"/>
			<lne id="112" begin="54" end="57"/>
			<lne id="113" begin="0" end="58"/>
			<lne id="114" begin="59" end="59"/>
			<lne id="115" begin="59" end="60"/>
			<lne id="116" begin="61" end="61"/>
			<lne id="117" begin="59" end="62"/>
			<lne id="118" begin="0" end="63"/>
			<lne id="119" begin="64" end="64"/>
			<lne id="120" begin="64" end="65"/>
			<lne id="121" begin="66" end="66"/>
			<lne id="122" begin="64" end="67"/>
			<lne id="123" begin="0" end="68"/>
			<lne id="124" begin="69" end="69"/>
			<lne id="125" begin="69" end="70"/>
			<lne id="126" begin="71" end="71"/>
			<lne id="127" begin="69" end="72"/>
			<lne id="128" begin="0" end="73"/>
			<lne id="129" begin="0" end="74"/>
		</linenumbertable>
		<localvariabletable>
			<lve slot="0" name="27" begin="0" end="74"/>
		</localvariabletable>
	</operation>
	<operation name="14">
		<context type="130"/>
		<parameters>
		</parameters>
		<code>
			<pusht/>
		</code>
		<linenumbertable>
			<lne id="131" begin="0" end="0"/>
		</linenumbertable>
		<localvariabletable>
			<lve slot="0" name="27" begin="0" end="0"/>
		</localvariabletable>
	</operation>
	<operation name="16">
		<context type="130"/>
		<parameters>
		</parameters>
		<code>
			<pusht/>
		</code>
		<linenumbertable>
			<lne id="132" begin="0" end="0"/>
		</linenumbertable>
		<localvariabletable>
			<lve slot="0" name="27" begin="0" end="0"/>
		</localvariabletable>
	</operation>
	<operation name="18">
		<context type="35"/>
		<parameters>
		</parameters>
		<code>
			<pusht/>
		</code>
		<linenumbertable>
			<lne id="133" begin="0" end="0"/>
		</linenumbertable>
		<localvariabletable>
			<lve slot="0" name="27" begin="0" end="0"/>
		</localvariabletable>
	</operation>
	<operation name="20">
		<context type="35"/>
		<parameters>
		</parameters>
		<code>
			<pushf/>
		</code>
		<linenumbertable>
			<lne id="134" begin="0" end="0"/>
		</linenumbertable>
		<localvariabletable>
			<lve slot="0" name="27" begin="0" end="0"/>
		</localvariabletable>
	</operation>
</asm>
